README.md

```diff
@@ -48,11 +48,14 @@ There are three main playbooks: server.yml (Infinispan Server), hyperfoil_contro
 * server.yml - Starts the Infinispan server container nodes with host networking enabled. Also uploads the `cache_name` named cache with the configuration in the `cache_file` file.
   * Default `cache_name` variable is `benchmark` and `cache_file` is `files/cache.xml` if you wish to change the name or xml file.
   * This playbook can also be ran with `-e operation=shutdown` to shutdown the inventoried servers
+  * The configured host group can be replaced by passing `-e server_hosts=<host1,host2>'
 * hyperfoil.yml - Playbook that encompasses starting the hyperfoil controler, agents and starts a run and then shuts them all down. Note the server must already be running and configured in the server hosts file.
 * hyperfoil_controller.yml - Sets up the hyperfoil controller.
   * This playbook can also be ran with `-e operation=shutdown` to shutdown the controller and agents if running
+  * The configured host group can be replaced by passing `-e hyperfoil_controller=host'
```

Comment:
```suggestion
  * The configured host group can be replaced by passing `-e hyperfoil_controller_host=host'
```

Reply:
Thanks, instead of just deleting the `s` I deleted the whole word 🤒